<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuWindow</class>
 <widget class="QMainWindow" name="MenuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MenuWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>551</width>
      <height>251</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="connectWindow">
     <widget class="QLineEdit" name="serverAddress">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>110</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Server Address :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>231</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Masukkan Alamat Server yang dituju</string>
      </property>
     </widget>
     <widget class="QToolButton" name="chooseSignupButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>160</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>DAFTAR</string>
      </property>
     </widget>
     <widget class="QToolButton" name="chooseLoginButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>160</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>LOGIN</string>
      </property>
     </widget>
     <widget class="QLabel" name="serverError">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>90</y>
        <width>311</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;error server&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="loginWindow">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Masukkan Username dan Password anda!</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Username :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>140</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Password  :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="usernameLogin">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>90</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordLogin">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>140</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="loginButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>190</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>LOGIN</string>
      </property>
     </widget>
     <widget class="QToolButton" name="backButtonOnLogin">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>190</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>KEMBALI</string>
      </property>
     </widget>
     <widget class="QLabel" name="passwordLoginError">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>120</y>
        <width>411</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;error pass&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="userLoginError">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>411</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;error user&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="signupWindow">
     <widget class="QLineEdit" name="usernameSignup">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>80</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordSignup">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>130</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="confirmPasswordSignup">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>170</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Username :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>130</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Password  : </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>170</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirm Password :</string>
      </property>
     </widget>
     <widget class="QToolButton" name="signupButton">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>210</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>DAFTAR</string>
      </property>
     </widget>
     <widget class="QToolButton" name="backButtonOnSignup">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>210</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>KEMBALI</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>531</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Masukkan data akun, panjang username dan password harus lebih dari 3 Karakter!</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>(Rahasiakan password anda!)</string>
      </property>
     </widget>
     <widget class="QLabel" name="passwordSignupError">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>110</y>
        <width>411</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;error pass&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="userSignupError">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>411</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;error user&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ChatWindow">
     <widget class="QListWidget" name="listUserWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>131</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Online Users :</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="chatTextEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>50</y>
        <width>301</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="chatMessage">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>230</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Chat Window :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>230</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Chat :</string>
      </property>
     </widget>
     <widget class="QToolButton" name="sendButton">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>230</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>SEND</string>
      </property>
     </widget>
     <widget class="QToolButton" name="logoutButton">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>50</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>LOGOUT</string>
      </property>
     </widget>
     <widget class="QLabel" name="loggedUser">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>511</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Login sebagai :</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
